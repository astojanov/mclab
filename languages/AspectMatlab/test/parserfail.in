aspect test

properties
count = 0;
end

methods
function x = getCount()
x = count;
end
end

patterns
arraysetb + set(*, *.b);
end

actions

countsetb : after arraysetb
count = count + 1;
end

end
end