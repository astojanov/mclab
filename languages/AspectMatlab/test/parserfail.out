[14, 11]  unexpected token PLUS
[14, 11]  try replacing with COLON
